02_Instructor_Setup.Rmd

```diff
@@ -0,0 +1,268 @@
+```
+
+# Overview 
+
+<br>
+
+The setup instructions are presented in two parts:
+
+- **Instructor Setup:** covers everything needed to set up yourself and any co-instructors or TAs to work on AnVIL.  Depending on your team and funding, *you may only need to do these steps once* (and then make updates on an as-needed basis for changes in team members or funding).
+- **Running the Class:** covers how to grant your students access to AnVIL.  These steps will need to be *repeated for each new offering of the course*.
+
+## What do I need to do?
+
+You may not need to do everything in the instructor setup section!
+
+This section contains instructions for setting up billing from scratch.  If your funding is being managed by a third party (e.g. through a funding mechanism such as [STRIDES](https://datascience.nih.gov/strides) or through your institution) you can skip some steps.  The table below provides guidance on which steps you will need to complete:
+
+| Task | Self-managed Funding | 3rd-party Funding |
+|:----|:---:|:----:| 
+|[Create Google Account](#instructor-google-account)|Yes|Yes|
+|[Set up Google Billing](#google-billing-account)|Yes|Probably No (ask your funding manager)|
+|[Create Instructor/TA Group](#instructor-group)| Only if you have co-instructors or TAs|Only if you have co-instructors or TAs|
+|[Set up Terra Billing Projects](#terra-billing-projects)|Yes|Probably Yes (ask your funding manager)|
+
+# Create Google account {#instructor-google-account}
+
+AnVIL uses [Terra](https://anvil.terra.bio/) to run analyses. Terra operates on Google Cloud Platform, so you’ll pay for all storage and analysis costs through a Google account linked to Terra. The costs are the standard Google Cloud Platform fees for storing and moving data as well as executing an analysis. These costs are passed along through Terra without any markup.
+
+## Create a Google Account {#google-account}
+
+Terra operates on Google Cloud Platform, so you will need a (free) Google account which will allow you to access the Terra platform to manage students and in-class analyses.
+
+Your Google account will also (1) allow you to manage billing yourself or (2) allow a Program Manager to do so on your behalf.
+
+```{r, child=c("_child_create_google_account.Rmd")}
+
+```
+
+## Sign in to Terra
+
+You need to sign into Terra to allow Billing Project managers to add you to projects and/or Workspaces. [Launch Terra](https://anvil.terra.bio/#workspaces), and you should be prompted to sign in with your Google account.
+
+You can always access Terra by going to [`anvil.terra.bio`](https://anvil.terra.bio/), or by clicking the link on the AnVIL home page.
+
+```{r, echo=FALSE, fig.alt='Screenshot of the AnVIL home page. The button to "launch Terra" is highlighted.'}
+ottrpal::include_slide("https://docs.google.com/presentation/d/1tyt8ifxD3y7QW3nN27jh02aPrBvNEo1GN_u2Ne_UngM/edit#slide=id.gde7783227e_0_31")
+```
+
+# Set up Google Billing (some users) {#google-billing-account}
```

Comment:
I've been trying to avoid special characters - they break on leanpub/coursera renders

Reply:
oh, good to know, thanks